Breakout/Entities/BallEntity.cs

```diff
@@ -1,6 +1,86 @@
-﻿namespace Breakout.Entities;
+﻿using DIKUArcade.Entities;
+using DIKUArcade.Graphics;
+using DIKUArcade.Math;
+using DIKUArcade.Physics;
 
-public class BallEntity
+namespace Breakout.Entites;
+
+public class BallEntity : Entity
 {
-    //abc
+    #region Properties and fields
+    // Field for ball's movement speed and direction
+    private float _speed;
+    private Vec2F _direction;
+    // Maximum speed for ball entities
+    private const float MaxSpeed = 0.3f;
+    
+    #endregion
+    
+    #region Constructor
+    // Constructor that intializes player's shape and image by using the base class constructor
```

Comment:
XML dokumentation i stedet for kommentarer, mener det er built-in feature i jetbrains rider når du laver /// så det mega nice^^